src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Sentence.kt

```diff
@@ -102,6 +102,17 @@ data class Sentence(
       )
   }
 
+  private fun sameSentenceTypesWithout(secondSentence: Sentence, vararg sentenceTypes: SentenceType): Boolean {
+    return this.sentenceTypesWithout(*sentenceTypes)
+      .containsAll(secondSentence.sentenceTypesWithout(*sentenceTypes)) &&
```

Comment:
A minor thing - but Kotlin has a function `.and(...)` which might make this even more readable than it already is.

Reply:
Ah I didn't know that thanks!